### ISSUE TITLE
Version 6.1.0 Waring overlayRef prop.

### ISSUE THREAD

**Issue by tidytrax** [ericgio/react-bootstrap-typeahead#783]
Hello, since the last update 6.1.0/6.1.1 I'm getting this warning, 
Already tried disable strict mode.
besides the warning no problems, its possible to disable this warning?
thanks.

```sh
react_devtools_backend.js:2655 Warning: React does not recognize the `overlayRef` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `overlayref` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

```
- Version 6.1.0, 6.1.1
- Vite 4.2.1
- Node 19.8.1
- React 18.2.0
```

**Comment by ericgio** [comment]
Thanks for the report! Fixed in v6.1.2

**Comment by tidytrax** [comment]
Many thanks!!!
